### ISSUE TITLE
KeyboardioFirmware bridge

### ISSUE THREAD

**Issue by algernon** [algernon/Akela#46]
It would be nice to have a library that builds on KeyboardioFirmware, and presents it as Akela APIs, so that the various other libraries like `Akela-TapDance` and `Akela-Unicode` could easily be made to work with KeyboardioFirmware, too.

All the necessary bits to support this should already be in KeyboardioFirmware.

**Comment by algernon** [comment]
This is in progress, but blocked on a few KeyboardioFirmware pull requests.

**Comment by algernon** [comment]
This is done on master. Needs testing and better examples, but that's a different issue/topic.

**Comment by eltang** [comment]
I was wondering why the bridge was necessary, but I see that the whole project is now centered around it. I would have liked to see this project become a complete firmware, though.

**Comment by algernon** [comment]
A part of me, too. But having two competing firmware for an already niche product is not smart. And while my design was neat (or at least, so I like to think), it was far from perfect, and could not be made as end-user friendly as the KeyboardioFirmware + Akela pair will be.

So, small libs it is, and I might attempt porting KeyboardioFirmware to other hardware instead.

**Comment by eltang** [comment]
In that case I'm not sure whether merging our projects is going to work out. I'm not sure it'd even be possible if Akela is limited by having to depend on KeyboardioFirmware.

**Comment by algernon** [comment]
Likely not possible, indeed.

**Comment by eltang** [comment]
Don't bother trying to make sense of my code then; you wouldn't be able to use it anyway.
